packages/router/__tests__/router-test.ts

```diff
@@ -10012,16 +10022,16 @@ describe("a router", () => {
       // Perform an invalid navigation to /b/child which should cancel all
       // pending deferred's since nothing is reused.  It should not call bChild's
       // loader since it's below the boundary but should call b's loader.
-      let formData = new FormData();
-      formData.append("file", new Blob(["1", "2"]), "file.txt");
       let B = await t.navigate("/b/child", {
         formMethod: "get",
-        formData,
+        formData: new FormData(),
       });
 
+      await B.loaders.bChild.reject(
+        new Response("broken", { status: 400, statusText: "Bad Request" })
+      );
+
       // Both should be cancelled
-      await aDfd.resolve("Nope!");
-      await aChildDfd.resolve("Nope!");
```

Comment:
I think we want to keep these - to ensure their resolution doesn't leak through

Reply:
Done